### PR TITLE
The Honkmother Destroys

### PR DESCRIPTION
**Pull request by LadyFowls**
Adds a Entertainer role aswell as alt titles as a merchent type role where its every so often and a stage area in the bar to be usable by them, along with some other gag items

https://inews.co.uk/news/science/why-nasa-is-on-the-lookout-for-good-clowns/
https://futurism.com/nasa-studying-mars-class-clown

There is a genuine need to lighten the game up to bring us closer together as a community and I think issues in the past aside, this would be worth looking into readding

### CODE REVIEW

**Comment by garnascus** [comment]
As headmin of aurora i grant this PR my full support. +1

The role is desperately needed. Though it might be more RP appropriate to make them alt-titles of an "actor" job or "entertainer"

**Comment by flimango** [comment]
-1. I feel like the clown role, while a classic ss13 meme, is just that; a meme. It doesn't fit our HRP setting to have an official clown:tm: walking around the station slipping people and making jokes. I think it would be step in the wrong direction for the setting and tone we work so hard to build.

**Comment by LadyFowls** [comment]
> -1. I feel like the clown role, while a classic ss13 meme, is just that; a meme. It doesn't fit our HRP setting to have an official clown™️ walking around the station slipping people and making jokes. I think it would be step in the wrong direction for the setting and tone we work so hard to build.

Is it a issue if its a alt title to a general Entertainer role?

**Comment by Russell454** [comment]
There was a brief chat in OOC about the RP, generally the consensus is people are going to meta the role and treat it differently from an OOC standpoint due to the history of clowns and mimes.

Like, I get the point of the PR, its just I don't think it fits well into the Aurora setting.

-1

**Comment by LadyFowls** [comment]
> There was a brief chat in OOC about the RP, generally the consensus is people are going to meta the role and treat it differently from an OOC standpoint due to the history of clowns and mimes.
> 
> Like, I get the point of the PR, its just I don't think it fits well into the Aurora setting.
> 
> -1

Please state how the pr could be improved or provide feedback, this isnt a place for voting

File: code/game/gamemodes/cult/narsie.dm [hunk 1]
```diff
@@ -389,7 +390,37 @@ var/global/list/narsie_list = list()
...
+	..()
+	if(announce)
+		to_world("<font size='15' color='red'><b>[uppertext(name)] HAS BEEN SUMMONED</b></font>")
+		to_world(sound('sound/music/clownmusic.ogg'))
+
+	narsie_spawn_animation()
+
+	log_and_message_admins("Honkmother has been spawned.", location = get_turf(loc))
+
+	spawn(10 SECONDS)
```
Comment (skull132) [inline code/game/gamemodes/cult/narsie.dm]: timer this ree.

**Comment by KingOfThePing** [comment]
I find this is an important change and long overdue. We have a stage, we have a bar, we even have an old theatre. Adding this role will make people maybe want to utilize this all. It's a good PR.

**Comment by Kaedwuff** [comment]
0f5fb20f0d105bb8f36b8a9b1833a755

I can feel my soul shriveling up and dying.  

While I'm not opposed to there being entertainer roles, I flat out refuse to accept us re-implementing clowns to the server, along with typo-filled piemaker machines that actually use the term 'bluespace' to describe how the pies are made.

We're supposed to be better than this.  Make it less silly and memey or don't do it.

**Comment by LadyFowls** [comment]
> 0f5fb20f0d105bb8f36b8a9b1833a755
> 
> I can feel my soul shriveling up and dying.
> 
> While I'm not opposed to there being entertainer roles, I flat out refuse to accept us re-implementing clowns to the server, along with typo-filled piemaker machines that actually use the term 'bluespace' to describe how the pies are made.
> 
> We're supposed to be better than this. Make it less silly and memey or don't do it. No pies and banana peels and stupid shit.

please provide feedback on how this can be improved or made better

**Comment by Kaedwuff** [comment]
> > 0f5fb20f0d105bb8f36b8a9b1833a755
> > I can feel my soul shriveling up and dying.
> > While I'm not opposed to there being entertainer roles, I flat out refuse to accept us re-implementing clowns to the server, along with typo-filled piemaker machines that actually use the term 'bluespace' to describe how the pies are made.
> > We're supposed to be better than this. Make it less silly and memey or don't do it. No pies and banana peels and stupid shit.
> 
> please provide feedback on how this can be improved or made better

If you take a moment to carefully reread the second paragraph and third paragraph, you will find suggestions on how to make this better.

**Comment by LadyFowls** [comment]
> > > 0f5fb20f0d105bb8f36b8a9b1833a755
> > > I can feel my soul shriveling up and dying.
> > > While I'm not opposed to there being entertainer roles, I flat out refuse to accept us re-implementing clowns to the server, along with typo-filled piemaker machines that actually use the term 'bluespace' to describe how the pies are made.
> > > We're supposed to be better than this. Make it less silly and memey or don't do it. No pies and banana peels and stupid shit.
> > 
> > 
> > please provide feedback on how this can be improved or made better
> 
> If you take a moment to carefully reread the second paragraph and third paragraph, you will find suggestions on how to make this better.

There is only two paragraphs, it doees not give information on why clowns should not be added or how to help them fit into the setting

**Comment by Kaedwuff** [comment]
> There is only two paragraphs, it doees not give information on why clowns should not be added or how to help them fit into the setting

Sigh.  Okay, I'll reiterate.
-Adding entertainers is fine.  Don't add clowns.  Don't add clown stuff, like piemakers and bikehorns and clown backpacks.  We took away clowns for a reason, they don't fit in a serious workplace.  It's one thing to have someone around to entertain people and lighten the mood, it's another to bring something in that belongs in a circus or children's birthday party.  I'm iffy on mimes, but they're at least slightly more classy than clowns, and could possibly work.
-Double check your work for grammar and spelling errors.  
-Don't add jokey descriptions to items, like implying something works by bluespace magic.

**Comment by LadyFowls** [comment]
> > There is only two paragraphs, it doees not give information on why clowns should not be added or how to help them fit into the setting
> 
> Sigh. Okay, I'll reiterate.
> -Adding entertainers is fine. Don't add clowns. Don't add clown stuff, like piemakers and bikehorns and clown backpacks. We took away clowns for a reason, they don't fit in a serious workplace. It's one thing to have someone around to entertain people and lighten the mood, it's another to bring something in that belongs in a circus or children's birthday party. I'm iffy on mimes, but they're at least slightly more classy than clowns, and could possibly work.
> -Double check your work for grammar and spelling errors.
> -Don't add jokey descriptions to items, like implying something works by bluespace magic.

The machines desc is a placeholder. However I will remove the clown and mime alt titles but leave the equipment present for them to use if they wish to, because they should have the option to entertain how they wish

**Comment by Kaedwuff** [comment]
I suppose that's reasonable.  If they become extremely obnoxious we can deal with them IC, without the baggage of SS13 expectations on what a 'clown' or 'mime' is supposed to do getting in the way.

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 2]
```diff
@@ -556,6 +556,36 @@
 	nutriment_desc = list("sweetness" = 3, "cookie" = 2)
 	bitesize = 1
 
+
+/obj/item/weapon/reagent_containers/food/snacks/creampie
```
Comment (Alberyk) [inline code/modules/reagents/reagent_containers/food/snacks.dm]: We already have banana pies in the code.

**Comment by Alberyk** [comment]
This need a forum thread.

**Comment by LadyFowls** [comment]
gonna close this for now

**Pull request closed without merging**